object/service.proto

```diff
@@ -688,3 +692,15 @@ message GetRangeHashResponse {
   // transmission.
   neo.fs.v2.session.ResponseVerificationHeader verify_header = 3;
 }
+
+// TODO: docs
+message ReplicationRequest {
+  // TODO: reserve 1st byte for forward compatibility
+  bytes object = 1;
```

Comment:
Can we have it be structured then? Like a proper `Object`? That's the one we send anyway, so if you have it serialized, that's your message, or not?

Reply:
yeah all messages are binary